AFMReader/jpk.py

```diff
@@ -42,6 +40,35 @@ def _jpk_pixel_to_nm_scaling(tiff_page: tifffile.tifffile.TiffPage, jpk_tags: di
     return px_to_nm * 1e9
 
 
+def get_tag_value(page, tag_name) -> Any:
```

Comment:
```suggestion
def get_tag_value(page: tifffile.TiffPage, tag_name: str) -> str | int | float:
```

Guessing most things returned will be string/integer/floats.

Reply:
Hi @ns-rse, I made this chanage however this is now causing an issue with pre-commit. It doesn't like that ` str | int | float` doesn't match the types of arguments that the tag values are used for, this has caused a chain of pre-commit errors.

Reply:
Resolved by using int() to explicitly make `n_slots ` an integer.

Reply:
Adding error catching made the function too complex for pre commit so separated a function to find the number of slots. Now passes pre commit.